dissect/target/plugins/os/windows/network.py

```diff
@@ -0,0 +1,383 @@
+    73: "ESCON",
+    74: "DLSW",
+    75: "ISDN_S",
+    76: "ISDN_U",
+    77: "LAP_D",
+    78: "IPSWITCH",
+    79: "RSRB",
+    80: "ATM_LOGICAL",
+    81: "DS0",
+    82: "DS0_BUNDLE",
+    83: "BSC",
+    84: "ASYNC",
+    85: "CNR",
+    86: "ISO88025R_DTR",
+    87: "EPLRS",
+    88: "ARAP",
+    89: "PROP_CNLS",
+    90: "HOSTPAD",
+    91: "TERMPAD",
+    92: "FRAMERELAY_MPI",
+    93: "X213",
+    94: "ADSL",
+    95: "RADSL",
+    96: "SDSL",
+    97: "VDSL",
+    98: "ISO88025_CRFPRINT",
+    99: "MYRINET",
+    100: "VOICE_EM",
+    101: "VOICE_FXO",
+    102: "VOICE_FXS",
+    103: "VOICE_ENCAP",
+    104: "VOICE_OVERIP",
+    105: "ATM_DXI",
+    106: "ATM_FUNI",
+    107: "ATM_IMA",
+    108: "PPPMULTILINKBUNDLE",
+    109: "IPOVER_CDLC",
+    110: "IPOVER_CLAW",
+    111: "STACKTOSTACK",
+    112: "VIRTUALIPADDRESS",
+    113: "MPC",
+    114: "IPOVER_ATM",
+    115: "ISO88025_FIBER",
+    116: "TDLC",
+    117: "GIGABITETHERNET",
+    118: "HDLC",
+    119: "LAP_F",
+    120: "V37",
+    121: "X25_MLP",
+    122: "X25_HUNTGROUP",
+    123: "TRANSPHDLC",
+    124: "INTERLEAVE",
+    125: "FAST",
+    126: "IP",
+    127: "DOCSCABLE_MACLAYER",
+    128: "DOCSCABLE_DOWNSTREAM",
+    129: "DOCSCABLE_UPSTREAM",
+    130: "A12MPPSWITCH",
+    131: "TUNNEL",
+    132: "COFFEE",
+    133: "CES",
+    134: "ATM_SUBINTERFACE",
+    135: "L2_VLAN",
+    136: "L3_IPVLAN",
+    137: "L3_IPXVLAN",
+    138: "DIGITALPOWERLINE",
+    139: "MEDIAMAILOVERIP",
+    140: "DTM",
+    141: "DCN",
+    142: "IPFORWARD",
+    143: "MSDSL",
+    144: "IEEE1394",
+    145: "IF_GSN",
+    146: "DVBRCC_MACLAYER",
+    147: "DVBRCC_DOWNSTREAM",
+    148: "DVBRCC_UPSTREAM",
+    149: "ATM_VIRTUAL",
+    150: "MPLS_TUNNEL",
+    151: "SRP",
+    152: "VOICEOVERATM",
+    153: "VOICEOVERFRAMERELAY",
+    154: "IDSL",
+    155: "COMPOSITELINK",
+    156: "SS7_SIGLINK",
+    157: "PROP_WIRELESS_P2P",
+    158: "FR_FORWARD",
+    159: "RFC1483",
+    160: "USB",
+    161: "IEEE8023AD_LAG",
+    162: "BGP_POLICY_ACCOUNTING",
+    163: "FRF16_MFR_BUNDLE",
+    164: "H323_GATEKEEPER",
+    165: "H323_PROXY",
+    166: "MPLS",
+    167: "MF_SIGLINK",
+    168: "HDSL2",
+    169: "SHDSL",
+    170: "DS1_FDL",
+    171: "POS",
+    172: "DVB_ASI_IN",
+    173: "DVB_ASI_OUT",
+    174: "PLC",
+    175: "NFAS",
+    176: "TR008",
+    177: "GR303_RDT",
+    178: "GR303_IDT",
+    179: "ISUP",
+    180: "PROP_DOCS_WIRELESS_MACLAYER",
+    181: "PROP_DOCS_WIRELESS_DOWNSTREAM",
+    182: "PROP_DOCS_WIRELESS_UPSTREAM",
+    183: "HIPERLAN2",
+    184: "PROP_BWA_P2MP",
+    185: "SONET_OVERHEAD_CHANNEL",
+    186: "DIGITAL_WRAPPER_OVERHEAD_CHANNEL",
+    187: "AAL2",
+    188: "RADIO_MAC",
+    189: "ATM_RADIO",
+    190: "IMT",
+    191: "MVL",
+    192: "REACH_DSL",
+    193: "FR_DLCI_ENDPT",
+    194: "ATM_VCI_ENDPT",
+    195: "OPTICAL_CHANNEL",
+    196: "OPTICAL_TRANSPORT",
+    243: "WWANPP",
+    244: "WWANPP2",
+}
+
+PriorityVlanTagEnum = {
```

Comment:
Also move this to the top but probably fine keeping it like a dict because of the string.